tests/build/multi-platform.go

```diff
@@ -432,13 +417,19 @@ var _ = framework.MultiPlatformBuildSuiteDescribe("Multi Platform Controller E2E
 	})
 })
 
-func createApplicationAndComponent(f *framework.Framework, testNamespace string) (component *appservice.Component, applicationName, componentName string) {
+func createApplicationAndComponent(f *framework.Framework, testNamespace, platform string) (component *appservice.Component, applicationName, componentName string) {
 	applicationName = fmt.Sprintf("multi-platform-suite-application-%s", util.GenerateRandomString(4))
 	_, err := f.AsKubeAdmin.HasController.CreateApplication(applicationName, testNamespace)
 	Expect(err).NotTo(HaveOccurred())
 
 	componentName = fmt.Sprintf("multi-platform-suite-component-%s", util.GenerateRandomString(4))
 
+	customBuildahRemotePipeline := os.Getenv(constants.CUSTOM_BUILDAH_REMOTE_PIPELINE_BUILD_BUNDLE_ENV + "_" + platform)
+	Expect(customBuildahRemotePipeline).ShouldNot(BeEmpty())
+	buildPipelineAnnotation := map[string]string{
+		"build.appstudio.openshift.io/pipeline": fmt.Sprintf(`{"name":"java-builder", "bundle": "%s"}`, customBuildahRemotePipeline),
```

Comment:
For consistency, this one should probably be `docker-build` as well

Reply:
AS in the previous case it doesn't matter, but will change to `buildah-remote-pipeline` as it used to be.